openedx/core/djangoapps/discussions/config/waffle.py

```diff
@@ -2,9 +2,9 @@
 This module contains various configuration settings via
 waffle switches for the discussions app.
 """
-
 from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
 
+WAFFLE_FLAG_NAMESPACE = "discussions"
 
 WAFFLE_NAMESPACE = 'discussions'
```

Comment:
Can `WAFFLE_NAMESPACE` be reused, is there a reason to create `WAFFLE_FLAG_NAMESPACE` ?